### PR TITLE
Role to generate reverseproxy facts from host and groupvars

### PR DESCRIPTION
**Pull request by Mr-Pi**
* task and script to generate proper reverseproxy facts from all group and hostvars
* documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+#### Primary
+| Option         | Type          | Default | Description                                                                                       | Required |
+|----------------|---------------|---------|---------------------------------------------------------------------------------------------------|:--------:|
+| served_domains | list of dicts |         | See [reverse_proxy](https://github.com/stuvusIT/reverse_proxy#served_domains) for possible values | Y        |
+
+
+## Example Playbook
+
+### Vars:
+`reverse_proxy_sever01.yml`:
```
Comment (SlothOfAnarchy) [inline README.md]: *server

File: README.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+```yml
+description: Reverse Proxy
+letsencrypt_email: fuu@example.com
+default_url: https://stuvus.uni-stuttgart.de
+domain_prefixes:
+  - www
+domain_suffixes:
+  - example.com
+interfaces:
```
Comment (SlothOfAnarchy) [inline README.md]: remove or populate interfaces

File: README.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+letsencrypt_email: fuu@example.com
+default_url: https://stuvus.uni-stuttgart.de
+domain_prefixes:
+  - www
+domain_suffixes:
+  - example.com
+proxy_domains:
+  - target_description: Webserver 1 for public usage ( testing )
+    target_host: web01
+	target_ip: 192.168.0.2
```
Comment (SlothOfAnarchy) [inline README.md]: indent correctly

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Comment by Mr-Pi** [comment]
please rereview

File: README.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+default_url: https://stuvus.uni-stuttgart.de
+domain_prefixes:
+  - www
+domain_suffixes:
+  - example.com
+proxy_domains:
+  - target_description: Webserver 1 for public usage ( testing )
+    target_host: web01
+    target_ip: 192.168.0.2
+	served_domains:
```
Comment (SlothOfAnarchy) [inline README.md]: Indent. Also, my other concerns have not been adressed.

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

**Comment by Mr-Pi** [comment]
Please rereview

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]
https://github.com/stuvusIT/reverse_proxy_mklist/pull/2/files#r140637993

https://github.com/stuvusIT/reverse_proxy_mklist/pull/2/files#r140637988

**Comment by Mr-Pi** [comment]
Okay and again

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**